RecordShow.java

```diff
@@ -0,0 +1,113 @@
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.util.Scanner;
+
+class RecordShow {
+    // start Recoding after inserting programme name and time id
+    int user_id;
+
+    void recordShow(Scanner inp, int user_id) throws Exception {
+        user_id = this.user_id;
+        PreparedStatement pst;
+        int rs;
+        ResultSet resultSet;
+        SQLQueries sql = new SQLQueries();
```

Comment:
No need to create new SQLQueries Object in every function
Just declare one in the class and assign it in the constructor